src/pages/Main/components/MainLocal.jsx

```diff
@@ -0,0 +1,190 @@
+import 'slick-carousel/slick/slick-theme.css';
+import { fontMix } from '../../../styles/mixin';
+
+const MainLocal = () => {
+  const [mainlocal, setMainlocal] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/mainlocaldata.json')
+      .then(res => res.json())
+      .then(data => {
+        setMainlocal(data);
+      });
+  }, []);
+  const settings = {
+    dots: false,
+    arrows: true,
+    infinite: true,
+    speed: 500,
+    slidesToShow: 4,
+    slidesToScroll: 1,
+    nextArrow: <NextArrow />,
+    prevArrow: <PrevArrow />,
+  };
+  return (
+    <div>
+      <Styledwrap>
+        <Local># 지역별 캠핑장</Local>
+        <StyledSlider {...settings}>
+          {mainlocal.map((item, index) => (
+            <SliderItem key={index}>
+              <ImageWrapper>
+                <img src={item.image} alt="각 지역에 있는 캠핑장" />
+              </ImageWrapper>
+              <ImageOverlay>
+                <Imagename>
+                  {item.name}
+                  <br />
+                </Imagename>
+                <Imagecount>{item.count} 개의 캠핑 상품.</Imagecount>
+                <TourButton>둘러보기</TourButton>
+              </ImageOverlay>
+            </SliderItem>
+          ))}
+        </StyledSlider>
+      </Styledwrap>
+    </div>
+  );
+};
+export default MainLocal;
+
+const NextArrow = ({ onClick }) => {
+  const nextArrowImage = '/images/Main/toleft.png';
+  return (
+    <NextArrowButton onClick={onClick} type="button">
+      <NextArrowImage src={nextArrowImage} alt="next" />
+    </NextArrowButton>
+  );
+};
+
+const PrevArrow = ({ onClick }) => {
+  const prevArrowImage = '/images/Main/toright.png';
+  return (
+    <PrevArrowButton onClick={onClick} type="button">
+      <PrevArrowImage src={prevArrowImage} alt="prev" />
+    </PrevArrowButton>
+  );
+};
+
+const NextArrowButton = styled.button`
+  position: absolute;
+  z-index: 1;
+  top: 45%;
+  left: 0;
+  width: 45px;
+  height: 45px;
+  border: none;
+  border-radius: 50%;
+  background-color: white;
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: 24px;
+  margin-top: 0px;
+`;
+
+const NextArrowImage = styled.img`
+  width: 100%;
+  height: 100%;
+`;
+
+const PrevArrowButton = styled.button`
+  position: absolute;
+  z-index: 1;
+  top: 45%;
+  right: 0;
+  width: 45px;
+  height: 45px;
+  border: none;
+  border-radius: 50%;
+  background-color: white;
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: 20px;
+  margin-top: 0px;
+`;
+
+const PrevArrowImage = styled.img`
+  width: 100%;
+  height: 100%;
+`;
+
+const Local = styled.h6`
+  ${fontMix(36, 'black')}
+  margin : 10px 0px 5px 0px;
+`;
+
+const StyledSlider = styled(Slider)`
+  .slick-slide div {
+    margin: 0 5px;
+    width: 100%;
+    height: auto;
+  }
+`;
+
+const SliderItem = styled.div`
+  margin: 0 10px;
+  margin-top: 100px;
+  position: relative;
+  padding-top: 30px;
+  padding-right: 30px;
+  padding-bottom: 15px;
```

Comment:
```suggestion
  padding: 30px 30px 15px 0;
```
여러개 스타일을 사용하는 것 보다 이렇게 한 줄로 작성하는 것이 좋겠네요!